### PR TITLE
[Docs] Fix #3129: Add swagger api documentation for remove_participant_team_from_challenge API

### LINKED ISSUE #3129: Add swagger api documentation for remove_participant_team_from_challenge API
### Description

Hosts can use EvalAI APIs to remove participant teams from an active challenge. The remove participant team from challenge API documentation is missing documentation on EvalAI [docs](http://eval.ai/api/docs/).

### Deliverables
Add documentation for using `remove_participant_team_from_challenge` API using swagger. An example of using swagger for API documentation is available [here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/jobs/views.py#L87). The documentation should provide a list of query and header params described briefly and an example.

### PR DESCRIPTION
**Pull request by afif-fahim**
Fixes #3129

### CODE REVIEW

**Comment by afif-fahim** [comment]
@Ram81 can you review this PR, please?

File: apps/participants/views.py [hunk 1]
```diff
@@ -13,6 +13,8 @@
 )
 from rest_framework.throttling import UserRateThrottle
 from rest_framework_simplejwt.authentication import JWTAuthentication
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
```
Comment (Ram81) [inline apps/participants/views.py:17]: @afif-fahim can you reorder imports. We arrange imports lexicographically

File: apps/participants/views.py [hunk 2]
```diff
@@ -456,6 +458,35 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
...
+            type=openapi.TYPE_NUMBER,
+            description="Participant Team Primary Key",
+            required=True,
+        ),
+    ],
+    operation_id="remove_participant_team_from_challenge",
+    responses={
+        status.HTTP_200_OK: openapi.Response(""),
+        status.HTTP_400_BAD_REQUEST: openapi.Response(
+            "{'error': 'Error message goes here'}"
```
Comment (Ram81) [inline apps/participants/views.py:483]: @afif-fahim can you replace the error message

File: apps/participants/views.py [hunk 3]
```diff
@@ -456,6 +458,35 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
...
 
 
+@swagger_auto_schema(
+    methods=["post"],
+    manual_parameters=[
+        openapi.Parameter(
+            name="challenge_pk",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_NUMBER,
+            description="Challenge Primary Key",
```
Comment (Ram81) [inline apps/participants/views.py:468]: change `Challenge Primary Key` to `Challenge id`

File: apps/participants/views.py [hunk 4]
```diff
@@ -456,6 +458,35 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
...
+            required=True,
+        ),
+        openapi.Parameter(
+            name="participant_team_pk",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_NUMBER,
+            description="Participant Team Primary Key",
```
Comment (Ram81) [inline apps/participants/views.py:475]: Change `Participant Team primary key` to `Participant team id`

File: apps/participants/views.py [hunk 5]
```diff
@@ -466,13 +497,9 @@ def remove_participant_team_from_challenge(
...
 
-    Arguments:
-        request {HttpRequest} -- The request object
-        challenge_pk {[int]} -- Challenge primary key
-        participant_team_pk {[int]} -- Participant team primary key
-
-    Returns:
-        Response Object -- An object containing api response
+    - Arguments:
+       - ``challenge_pk``: Primary key for the challenge from which a participant team is to be removed
```
Comment (Ram81) [inline apps/participants/views.py:501]: @afif-fahim can you please rever this

**Review by Ram81 — CHANGES_REQUESTED** [review]
@afif-fahim can you also share a screenshot from the `api/docs` page for these changes

**Comment by afif-fahim** [comment]
@Ram81 here is the api/docs preview after changes

image

File: apps/participants/views.py (hunk 1 above)
Comment (afif-fahim) [reply apps/participants/views.py:17]: image
Is this okay now? I am confused sorry! I followed `challenges/views.py`, `jobs/views.py` to arrange the imports.

File: apps/participants/views.py (hunk 3 above)
Comment (afif-fahim) [reply apps/participants/views.py:468]: Done

File: apps/participants/views.py (hunk 4 above)
Comment (afif-fahim) [reply apps/participants/views.py:475]: Done

File: apps/participants/views.py (hunk 2 above)
Comment (afif-fahim) [reply apps/participants/views.py:483]: @Ram81 I think swagger allows a single response schema for each response code, but this API has two response schemas for 400 response code
image
How should I replace the error message now?

File: apps/participants/views.py (hunk 5 above)
Comment (afif-fahim) [reply apps/participants/views.py:501]: Done

File: apps/participants/views.py (hunk 1 above)
Comment (Ram81) [reply apps/participants/views.py:17]: these two imports should go on line 3-4

File: apps/participants/views.py (hunk 3 above)
Comment (Ram81) [reply apps/participants/views.py:468]: @afif-fahim this is not updated. Did you push the commit?

File: apps/participants/views.py (hunk 2 above)
Comment (Ram81) [reply apps/participants/views.py:483]: You can mention the `Team has not participated in the challenge` response

File: apps/participants/views.py (hunk 5 above)
Comment (Ram81) [reply apps/participants/views.py:501]: can you update the PR

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: apps/participants/views.py (hunk 3 above)
Comment (afif-fahim) [reply apps/participants/views.py:468]: Sorry I have not! I am updating the PR in a moment.

File: apps/participants/views.py (hunk 5 above)
Comment (afif-fahim) [reply apps/participants/views.py]: @Ram81 please check the PR now.

File: apps/participants/views.py [hunk 6]
```diff
@@ -466,13 +498,9 @@ def remove_participant_team_from_challenge(
...
     API to remove the participant team from a challenge
 
-    Arguments:
-        request {HttpRequest} -- The request object
-        challenge_pk {[int]} -- Challenge primary key
-        participant_team_pk {[int]} -- Participant team primary key
-
-    Returns:
-        Response Object -- An object containing api response
+    - Arguments:
```
Comment (Ram81) [inline apps/participants/views.py:501]: @afif-fahim please revert the changes to docstrings

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: apps/participants/views.py (hunk 6 above)
Comment (afif-fahim) [reply apps/participants/views.py]: @Ram81 reverted the changes. Preview after reverting:
image

File: apps/participants/views.py [hunk 7]
```diff
@@ -1,5 +1,8 @@
 from django.contrib.auth.models import User
 
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+
```
Comment (Ram81) [inline apps/participants/views.py:5]: remove newline

File: apps/participants/views.py [hunk 8]
```diff
@@ -456,6 +459,35 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
...
 
 
+@swagger_auto_schema(
+    methods=["post"],
+    manual_parameters=[
+        openapi.Parameter(
+            name="challenge_pk",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_NUMBER,
+            description="Challenge ID",
```
Comment (Ram81) [inline apps/participants/views.py:469]: Change `Challenge ID` to `Challenge pk`

File: apps/participants/views.py [hunk 9]
```diff
@@ -456,6 +459,35 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
...
+            required=True,
+        ),
+        openapi.Parameter(
+            name="participant_team_pk",
+            in_=openapi.IN_PATH,
+            type=openapi.TYPE_NUMBER,
+            description="Participant Team ID",
```
Comment (Ram81) [inline apps/participants/views.py:476]: Change `Participant Team ID` to `Participant team pk`

**Review by Ram81 — CHANGES_REQUESTED** [review]

**Comment by afif-fahim** [comment]
@Ram81 updated the PR!

**Comment by Ram81** [comment]
@afif-fahim can you share updated screenshot of the docs page

**Comment by afif-fahim** [comment]
@Ram81, Updated screenshot of doc:
image

**Review by Ram81 — APPROVED** [review]
LGTM 👍
